aws_marketplace/using_model_packages/data_quality_monitoring/monitoring_data_quality_of_models.ipynb

```diff
@@ -0,0 +1,776 @@
+{
```

Comment:
<p>This paragraph makes more sense if you have described what are the constraints beforehand</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2024/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>